src/main/java/org/stellar/sdk/FeeBumpTransaction.java

```diff
@@ -202,4 +201,21 @@ public FeeBumpTransaction build() {
           this.accountConverter, this.feeAccount, this.baseFee, this.innerTransaction);
     }
   }
+
+  @Override
+  public boolean equals(Object object) {
+    if (this == object) {
+      return true;
+    }
+    if (object == null || getClass() != object.getClass()) {
+      return false;
+    }
+    FeeBumpTransaction that = (FeeBumpTransaction) object;
+    return Objects.equals(toEnvelopeXdr(), that.toEnvelopeXdr());
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(toEnvelopeXdr());
```

Comment:
same question here, can the `hash()` method from AbstractTransaction be used instead?

Reply:
now it based on `signatureBase()`.